client/src/App.jsx

```diff
@@ -1,35 +1,34 @@
 import { useState, useEffect } from 'react';
+//'switch' has been replaced with 'routes'
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
```

Comment:
```suggestion
import { Route, Routes } from 'react-router-dom';
```
move `import { BrowserRouter as Router } from 'react-router-dom'` to your `./main.jsx`

Reply:
Done!